docs/source/_templates/footer.html

```diff
@@ -35,6 +35,12 @@
   <div>
     <a href="https://www.firstinspires.org/about/privacy-policy">Privacy Policy</a>
   </div>
+  <div>
+    <a href="/tos/tos.html">Terms of Service</a>
+  </div>
+  <div>
+    <a href="https://www.pavesuite.com/FIRST/PublicPortal/HomePage">Report an Incident</a>
```

Comment:
Didn't you say on Thursday you didn't want this on ftc-docs?  Did you change your mind?

Reply:
I totally did say that. However, Chris Rake overrode me in the Teams chat we had after our meeting when he said:
![image](https://user-images.githubusercontent.com/5846611/208326353-8a08f8a3-4f0e-48d8-b7e0-329697af207b.png)

Therefore, we should add it.